packages/backend/src/apps/m365-excel/actions/create-table-row/index.ts

```diff
@@ -135,12 +145,36 @@ const action: IRawAction = {
       return
     }
 
+    // Sanity check user's config.
+    const seenColumnNames = new Set<string>()
+    for (const val of columnValues) {
+      const currColumnName = String(val.columnName)
+
+      if (seenColumnNames.has(currColumnName)) {
+        throw new Error(
+          `Trying to write 2 different values to the same column (${currColumnName})`,
+        )
```

Comment:
nit: use `generateStepError` to nicely return this error

Reply:
edit: use new StepError once malcolm's pr is merged

Reply:
i was thinking you can merge ur excel PRs first and i can help to resolve the merge conflict on my PR later on

Reply:
nvm i refactored already no worries